workbench/src/main/investUsageLogger.js

```diff
@@ -13,12 +13,12 @@ const PREFIX = 'api';
 export default function investUsageLogger() {
   const sessionId = crypto.randomUUID();
 
-  function start(modelPyName, args, port) {
+  function start(modelID, args, port) {
```

Comment:
This is one more functional change: for usage logging, recording model IDs rather than python package names. I'm curious what everyone thinks about the impact of this change and if it's okay to go ahead with

Reply:
We'd need to update the corresponding Cloud Run function, and retain backwards compatibility for a while.

Reply:
Oh you're right, I forgot it was being translated! I think we can keep using python package names for now, and maybe change things later if we do more with usage logging.

If we do want to log plugin usage accurately, we might need to start logging the git source or something more specific - since multiple plugins, or versions of a plugin, could have the same ID and pyname. I'll make a separate issue for that.